<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DegenPrimerMain</class>
 <widget class="QMainWindow" name="DegenPrimerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DegenPrimer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DegenPrimerUI.qrc">
    <normaloff>:/degen_primer.svg</normaloff>:/degen_primer.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="mainTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="configTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Analysis options</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="terminalSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="configWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="3">
             <widget class="QPushButton" name="analyseButton">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Analyse primers&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Analyse</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>37</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="resetButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QScrollArea" name="configScrollArea">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="configFormWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>740</width>
                 <height>528</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="configForm"/>
              </widget>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="abortButton">
              <property name="text">
               <string>Abort</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>analyseButton</zorder>
           <zorder>horizontalSpacer</zorder>
           <zorder>resetButton</zorder>
           <zorder>configScrollArea</zorder>
           <zorder>abortButton</zorder>
          </widget>
          <widget class="QPlainTextEdit" name="terminalOutput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Terminal output&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="DegenPrimerUI.qrc"/>
 </resources>
 <connections/>
</ui>
